src/camera/Camera.js

```diff
@@ -764,6 +752,27 @@ FORGE.Camera.prototype._setYaw = function(value, unit)
 };
 
 /**
+ * Compute the yaw boundaries with yaw min and yaw max.
+ * @method FORGE.Camera#_getYawBoundaries
+ * @return {CameraBoundaries} Returns the min and max yaw computed from the camera configuration and the view limits.
+ * @private
+ */
+FORGE.Camera.prototype._getYawBoundaries = function()
+{
+    var min = this._yawMin;
+    var max = this._yawMax;
+    var view = this._viewer.renderer.view.current;
+
+    if (view !== null)
+    {
+        min = Math.max(view.yawMin, min);
+        max = Math.min(view.yawMax, max);
+    }
+
+    return { min: /** @type {number} */ (min), max: /** @type {number} */ (max) };
```

Comment:
Unnecessary if `_yawMin` is declared as `{number}` instead of `{?number}`, which is correct too (`_yawMin` is never set to null).